src/modules/companies/controller.js

```diff
@@ -44,4 +50,39 @@ const getReturns = async (request, h) => {
   return returns.map(rowMapper);
 };
 
+/**
+ * Gets a company for the specified ID
+ */
+const getCompany = async (request, h) => {
+  const { companyId } = request.params;
+  const company = await crmConnectors.companies.getCompany(companyId);
+  if (company) return company;
+  throw Boom.notFound(`Company ${companyId} not found`);
```

Comment:
I think we've been generally returning Boom errors rather than throwing (at least recently...)